__tests__/plugins.test.ts

```diff
@@ -0,0 +1,236 @@
+import { performance } from 'perf_hooks';
+
+import { jest } from '@jest/globals';
+import { number } from '@typeofweb/schema';
+
+import { createApp, createPlugin } from '../src';
+import { wait } from '../src/utils/utils';
+
+declare module '../src' {
+  interface TypeOfWebServerMeta {
+    readonly myPlugin: {
+      readonly someValue: number;
+      getUserById(id: string): Promise<readonly number[]>;
+    };
+  }
+}
+
+describe('plugins', () => {
+  it('should return function when cache is used', async () => {
+    const plugin = createPlugin('myPlugin', (_app) => {
+      return {
+        server(_server) {
+          return {
+            someValue: 42,
```

Comment:
Why do you add this field to this object?

Reply:
for fun